app.json

```
@@ -21,7 +21,8 @@
       "adaptiveIcon": {
         "foregroundImage": "./src/assets/app/adaptive-icon.png",
         "backgroundColor": "#ffffff"
-      }
+      },
+      "package": "com.anonymous.MakeSense"
```

I don't know for which purposes you've added it here, but if it is necessary, rename it to
```suggestion
"package": "com.xenous.makesense"
```
or just remove this line